### PR TITLE
Feature/implement partner section

### PR DESCRIPTION
**Pull request by quanghuy1799**
task: https://trello.com/c/vhYNl0oz

### CODE REVIEW

File: practice-two/src/css/main.css [hunk 1]
```diff
@@ -60,7 +72,7 @@
   }
 }
 
-@media screen and (min-width: 1440px) {
+@media screen and (min-width: 1025px) {
```
Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css:75]: quite confused, I will ping you to discuss this one

Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css]: Will discuss about 1440px vs 1025px

File: practice-two/src/css/main.css [hunk 2]
```diff
@@ -34,8 +35,19 @@
...
+  flex-wrap: wrap;
+  gap: 27px;
+  padding: 32px 0;
+}
+.partner-logo {
+  height: 20px;
+}
+@media screen and (min-width: 481px) {
   .container {
+    min-width: 896px;
```
Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css]: Quite confused, I will ping you to discuss this one

Comment (quanghuy1799) [reply practice-two/src/css/main.css]: i was fix it!!!

**Pull request merged**